src/components/onboarding/steps/Step2.vue

```diff
@@ -12,8 +12,8 @@
     <p>
       Fund transfers are instant, with near-zero processing fees.
       Censorship-proof: there are no third-party intermediaries to ban creators from sending and
-      receiving payments, and records are immortalized on æternity blockchain. Learn more by our
-      Tutorial & FAQ
+      receiving payments, and records are immortalized on æternity blockchain. <a href="https://youdonotneedacapetobeahero.com/#/create-profile" target="_blank">Learn more by our
```

Comment:
router?

Reply:
Its intentionally by designers request like this - so you don't leave the wizzard.

Reply:
I guess we can use something like this to avoid bugs after domain migration
```
<router-link to="/create-profile" v-slot="{ href }"><a :href="href" "target=_blank">Tutorial & FAQ</a></router-link>
```